With config functionality added. (#9011)
silverstripe/silverstripe-framework: docs/en/02_Developer_Guides/04_Configuration/00_Configuration.md

That is, the fragment will be included if all Only rules match, except if all Except rules match.
</div>

> ## Unit tests

Reviewer: Thanks for documenting this so well @mfendeksilverstripe, great addition! :)

Sometimes, it's necessary to change a configuration value in your unit tests.

> One of the ways to do this is to use the `withConfig` method.

Reviewer 2: ```suggestion
One way to do this is to use the `withConfig` method.
```

> This is very handy especially when using data providers.

Reviewer 2: ```suggestion
This is especially handy when using data providers.
```

Example below shows one unit test using a data provider.
This unit test changes configuration before testing functionality.
The test will run three times, each run with different configuration value.
Note that the configuration change is active only within the callback function.

> ```

Reviewer 2: ```suggestion
```php
```
